src/main/java/com/traverse/storage/models/CollectionSearchCriteria.java

```diff
@@ -0,0 +1,45 @@
+package com.traverse.storage.models;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+
+/**
+ * @author isfar
+ *
+ * This is the collection search criteria class. This is where devs can pass certain fields to query the collection
+ * This is standardized, so should work for all databases, i.e. neo4j or dynamodb.
+ *
+ */
+public class CollectionSearchCriteria {
+
+    /**
+     * This field is for text index search. Using user collection as an example,
+     * if a user types "Isfar" in search input box, this will search for Isfar
+     * in firstName, lastName, pfpURL, and username columns.
+     * To create a text index search in a neo4j db, run a command similar to this, which is used for user collection
+     * CREATE FULLTEXT INDEX textIndex FOR (n:User) ON EACH [n.firstName, n.lastName, n.pfpUrl, n.username];
+     */
+    @Getter
```

Comment:
Why not set a lombok anotation at the class level for all setters and getters? To keep it consistent throughout the apis its basically the same.